docs/security.md

```diff
@@ -0,0 +1,120 @@
+# Security best practices
+
+## Secrets detection
+
+It is far too easy to accidentally leak secrets on public repositories by
+inadvertently committing them in variables or configuration files after just
+doing a quick test during development.
+
+To prevent this kind of issues, it's highly suggested to integrate a solution
+like [Yelp/detect-secrets](https://github.com/Yelp/detect-secrets) in the
+standard development workflow.
+
+### First setup
+
+As a prerequisite,
+[install](https://github.com/Yelp/detect-secrets#installation) the
+`detect-secrets` CLI.
+
+The first step is to create the secrets baseline, that is just a JSON file
+holding all the settings and references all the potential secrets that have been
+detected in your codebase with:
+
+```sh
+detect-secrets scan > .secrets.baseline
+git add .secrets.baseline
+```
+
+Then you can start auditing all the detected secrets in your codebase with:
+
+```sh
+detect-secrets audit .secrets.baseline
+```
+
+For each detected secret, it will ask you if that secret is really meant to be
+present in the codebase or not:
+
+* Replying with Yes, will whitelist that entry as a non-secret (`is_secret`
+  field will be `false`)
+* Replying with No, will mark that entry as a secret that is meant to be removed
+  in the future (`is_secret` field will be `true`)
+* Replying with Skip, will skip to the next secret without marking the current
+  secret
+
```

Comment:
Can you add here a link to the secret skip section so people can choose to have a dummy password for their repo?
And also linking another section to exclude files based on path (with appropriate warning).

Reply:
not sure to have got everything you asked for, but I've added a link to the exclusion of secrets here and tried to clarify which is the purpose of the skip action while running the auditing command